ddtrace/ext/ci.py

```diff
@@ -249,6 +251,14 @@ def extract_bitbucket(env):
 def extract_buildkite(env):
     # type: (MutableMapping[str, str]) -> Dict[str, Optional[str]]
     """Extract CI tags from Buildkite environ."""
+    # Get alL keys which start with BUILDKITE_AGENT_META_DATA_x
+    node_label_list = []  # type: list[str]
+    BUILDKITE_AGENT_META_DATA_PREFIX = "BUILDKITE_AGENT_META_DATA_"
```

Comment:
Very small nit (just to show I actually read this PR): `BUILDKITE_AGENT_META_DATA_PREFIX` looks like a constant but it's just a variable in this function, so I'd make it lowercase.